hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -115,6 +121,21 @@ private MessageType getTableParquetSchemaFromDataFile() {
     }
   }
 
+  private MessageType getMessageType(String filePath) throws IOException {
```

Comment:
nit: rename to `readSchemaFromBaseFile`?

Reply:
Okay changed.